www/hooks/fetchModule.ts

```diff
@@ -0,0 +1,18 @@
+import useFetchProject from "./fetchProject";
+
+const useFetchModule = (projectName: string, moduleName: string) => {
+  const { project, projectJsonUrl, error, isLoading } =
+    useFetchProject(projectName);
+
+  return {
+    isLoading,
+    error,
+    project,
+    projectJsonUrl,
+    module: project
+      ? project.modules.find((mod) => mod.name === moduleName)
```

Comment:
Is the entire purpose of this hook this `find` call? Is that worth it?

Reply:
Its purpose is to provide a consistent interface for getting all levels of package information. Getting the project and the symbols is done via hooks (with proper logic encapsulated) and thus I feel that providing a hook for the middle tier of information as well is the consistent behavior.